attributes/default.rb

```diff
@@ -5,44 +5,20 @@
 
 default['qubit_bamboo']['log_path'] = '/var/log/bamboo-server.log'
 
-default['qubit_bamboo']['config']['marathon_port'] = 8080
-default['qubit_bamboo']['config']['marathon_hosts'] = ["http://localhost:#{node['qubit_bamboo']['config']['marathon_port']}"]
+default['qubit_bamboo']['marathon_port'] = 8080
+default['qubit_bamboo']['config']['Marathon']['Endpoint'] = ["http://localhost:#{node['qubit_bamboo']['marathon_port']}"].join(',')
```

Comment:
it needs to be written in the form of a string containing all the items with a , as a sperator